sdk/src/androidTest/java/com/mapbox/maps/CameraAnimationsPluginTest.kt

```diff
@@ -451,6 +451,36 @@ class CameraAnimationsPluginTest : BaseAnimationMapTest() {
     }
   }
 
+  @Test
+  fun testEaseToStartDelay() {
+    val targetBearing1 = 5.0
+    val targetBearing2 = 10.0
+    val cameraOptions1 = CameraOptions.Builder().bearing(targetBearing1).build()
+    val cameraOptions2 = CameraOptions.Builder().bearing(targetBearing2).build()
+    val expectedValues = mutableSetOf(targetBearing1, targetBearing2)
+    val updatedValues = mutableListOf<Double>()
+
+    val latch = CountDownLatch(2)
+    val cameraAnimationPlugin = mapView.camera
+    cameraAnimationPlugin.addCameraBearingChangeListener {
+      Logger.i(TAG, "onChanged $it")
+      updatedValues.add(it)
+      latch.countDown()
+    }
+    mainHandler.post {
+      cameraAnimationPlugin.easeTo(cameraOptions1, mapAnimationOptions { startDelay(0) })
+      cameraAnimationPlugin.easeTo(cameraOptions2, mapAnimationOptions { startDelay(500) })
+      // Only the first animation is started.
+      assertEquals(1, updatedValues.size)
+      assertEquals(targetBearing1, updatedValues.first(), 0.01)
```

Comment:
imo very non-reliable check + duration is not specified, it's not clear how this behaves in general
let's assign different owners to first `easeTo` and second one and check that
1. when run time is < 500 - first `easeTo` is running
2. when run time is > 500 - second `easeTo` is running, first is cancelled 
3. after second one is finished - verify we are located at end camera pos of second `easeTo`